.env.dev

```diff
@@ -4,4 +4,8 @@ DB_HOST = 'localhost'
 DB_PORT = 5432
 DB_USERNAME = 'msgcsuser'
 DB_PASSWORD = 'msgcspass'
-DB_NAME = 'msgcsdb'
\ No newline at end of file
+DB_NAME = 'msgcsdb'
+
+JWT_SECRET = 'jwt257sekretk923'
+
```

Comment:
**issue:** be very careful in the future so you won't publish secrets that you use as environment variables on git. It's very hard to clear the git history afterwards for a PR and you will suffer